packages/flutter/lib/src/widgets/drag_target.dart

```diff
@@ -400,11 +406,14 @@ class Draggable<T extends Object> extends StatefulWidget {
   MultiDragGestureRecognizer createRecognizer(GestureMultiDragStartCallback onStart) {
```

Comment:
This is called from initState and never again. What if supportedDevices is changed to something else during the lifetime of the widget?